core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletMetadata.java

```diff
@@ -98,11 +104,10 @@
   }
 
   public enum ColumnType {
-    LOCATION,
+    LOCATION, // includes all types in LocationType + SUSPEND
     PREV_ROW,
     OLD_PREV_ROW,
     FILES,
-    LAST,
```

Comment:
If there's any chance this is serialized, removing the enum changes the ordinals and could break things. Might be safer to mark it as deprecated and add a comment that it is no longer in use.

Reply:
I don't think so.  This enum is used to tell the class what column families to fetch in the scanner created in `TabletsMetadata`.  There is also a check `ensureFetched(ColumnType col)` that is called on the get methods that will throw an error if you didn't fetch the column family associated with that type.

Reply:
As long as it's just used internally, and never serialized or sent over the wire, it's fine.